<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalEventSeries_Widget</class>
 <widget class="QWidget" name="DigitalEventSeries_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Total Events: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="total_events_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="add_event_button">
         <property name="text">
          <string>Add Event</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_event_button">
         <property name="text">
          <string>Remove Event</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Section" name="export_section" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="exportSectionLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="filenameLayout">
        <item>
         <widget class="QLabel" name="label_filename">
          <property name="text">
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filename_edit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="exportTypeLayout">
        <item>
         <widget class="QLabel" name="label_export_type">
          <property name="text">
           <string>Export Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="export_type_combo">
          <item>
           <property name="text">
            <string>CSV</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="exportTypeSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stacked_saver_options">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="CSVEventSaver_Widget" name="csv_event_saver_widget"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSVEventSaver_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/DigitalTimeSeries/CSV/CSVEventSaver_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Section</class>
   <extends>QWidget</extends>
   <header>Collapsible_Widget/Section.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
